bilingual-module/README

```diff
@@ -31,44 +31,51 @@ Compiling
 
 Given the requirements being installed, you should be able to just run:
 
-    $ ./configure
-    $ make
-    # make install
+```console
```

Comment:
This ends up making the make install look like a comment, right? Not great :\

Reply:
No, the `make install` looks just fine (below is exactly the same markdown used in the readme):

```console
$ ./configure
$ make
# make install
```

Reply:
Ah, right you switched to console. It doesn't look any different without the hint though?

Reply:
No, this is how it looks like without the language hint:

```
$ ./configure
$ make
# make install
```

I guess we could remove the hint, but having it there is a bit clearer.

Reply:
I don't see any difference...?

![image](https://user-images.githubusercontent.com/4211838/48293094-4c09ab80-e432-11e8-8512-046c8f18db41.png)

Reply:
It does looks exactly the same.
When I said that having the hint is a bit clearer, I meant that if I read the source of the readme, I'll immediately know the code block is comprised of bash commands.

Reply:
Hmm, they're not bash per se. But, yeah. Can we make them all `console` for consistency?

Reply:
We can. The reason some blocks have the `console` hint while others have the `bash` hint is that the `bash` hint adds syntax highlighting to the commands, which looks a bit better IMO compared to the plain `console` block. I've used `bash` wherever possible (i.e. block which don't contain `#` commands or command outputs).

Reply:
Where does bash add syntax highlighting here:

https://github.com/sushain97/test-eng#help-and-support

?